README.md

```diff
@@ -302,6 +303,28 @@ script:
       bash <(curl -s https://codecov.io/bash) -f lcov.info;
 ```
 
+#### grcov with GitLab
+
+Here is an example of a `.gitlab-ci.yml` file for source-based coverage, using
+GitLab's [test coverage visualization feature](https://docs.gitlab.com/ee/user/project/merge_requests/test_coverage_visualization.html):
+
+```yaml
+test:
+  image: rust:latest
+  variables:
+    RUSTC_BOOTSTRAP: 1
+    RUSTFLAGS: -Zinstrument-coverage
+  script:
+    - rustup component add llvm-tools-preview
+    - curl -sSL https://github.com/mozilla/grcov/releases/latest/download/grcov-v0.8.6-x86_64-unknown-linux-gnu.tar.gz | tar -xz
```

Comment:
nit: These steps could also be part of a [`before_script:`](https://docs.gitlab.com/ee/ci/yaml/#before_script) section, as it prepares the environment for the actual coverage data collection.